usr/share/rear/backup/NETFS/default/41_save_capabilities.sh

```diff
@@ -0,0 +1,11 @@
+# save all found capapilities to file 
+
+if [ "$BACKUP_CAP" == "y" ] ; then
+	Log "Saving capabilities to File." 
+	if which getcap >/dev/null 2>&1 ; then
+		getcap -r / 2>/dev/null | grep -v $ISO_DIR > $VAR_DIR/recovery/capabilities
+	else
+		Log "getcap binary not found."
```

Comment:
Use variable `NETFS_RESTORE_CAPABILITIES` instead of `which getcap`. You were right `getcap` was already checked before if it was available via `REQUIRED_PROGS`